### ISSUE TITLE
Question about RTMP

### ISSUE THREAD

**Issue by nvcken** [GuilhermeMedeiros/react-native-videocore#3]
```
VideoCore.startStreaming('rtmp://104.155.71.82:1935/live' ...
```

Could you please explain this `rtmp://104.155.71.82:1935/live`, the pattern of mine is
iOS device use camera to feed video (react-native-videocore ) to --> Server -> server serves broadcast to multi iOS devices via HLS playlist?
- 104.155.71.82:1935 : is this IP & port of server ??
- Could I use RTSP instead of RTMP ??
  Thanks

**Comment by olivierlesnicki** [comment]
- 104.155.71.82 is IP and 1935 is port (replace them with your actual server IP and PORT)
- You need to use RTMP

**Comment by nvcken** [comment]
ok thanks.
Could you please help me any hints about RTSP ?

**Comment by olivierlesnicki** [comment]
Note yet supported by VideoCore (see https://github.com/jgh-/VideoCore/issues/95) which is the underlaying library used by react-native-videocore.

**Comment by ghost** [comment]
@nvcken Chiming in as I created that issue a while back. I was unable to open-source the code as it's owned by a former employer, although it isn't _too_ hard to adapt `VCSimpleSession` to use a protocol supported by [Live555's liveMedia](http://www.live555.com/liveMedia/). Really you'll have to just override the last two components in the streaming pipeline, and use timing information compatible with Live555's public API. From what I remember, you'll have to:
-  implement your own version of `RTMPSession`, `RTMPTypes`, `H264Packetizer`, and `AACPacketizer` using Live555
- convert `CMTime` to a format Live555 expects
- parse the stream to get the `sps` and `pps` NAL units, as well as the `asc` to create an SDP file.
- Make a cocoapod/package for Live555 as a static library

Basically you write a shim for VideoCore or add classes to the library.

**Comment by bhaveshtandel** [comment]
@Zintinio Hi, I am trying upload video using RTSP. How do i import Live555 library in my project using cocoapod? any steps which i need to follow? i am using "live555-latest.tar.gz" file from Source code.(http://www.live555.com/liveMedia/public/)

**Comment by ghost** [comment]
Compile it and make a pod or use the outdated version on my github page.
